app/src/main/res/layout/loyalty_card_edit_activity.xml

```diff
@@ -239,7 +245,152 @@
+
+                    <TextView
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginTop="16dp"
+                        android:fontFamily="sans-serif-medium"
+                        android:text="@string/reminders"
+                        android:textSize="22sp" />
+
+                </LinearLayout>
+
+                <View
+                    android:id="@+id/divider"
+                    android:layout_width="1dp"
+                    android:layout_height="match_parent"
+                    android:layout_marginTop="3dp"
+                    android:layout_marginStart="8dp"
+                    android:layout_marginEnd="8dp"
+                    android:background="?android:attr/listDivider" />
+
+                <LinearLayout
+                    android:id="@+id/showRemindersLayout"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:paddingHorizontal="@dimen/inputPadding"
+                    android:paddingTop="@dimen/inputPadding"
+                    android:layout_marginTop="5dp"
+                    android:orientation="horizontal">
+
+                    <!-- Show reminders -->
+
+                    <androidx.cardview.widget.CardView
+                        android:id="@+id/reminderCardView"
+                        android:layout_width="0dp"
+                        android:layout_height="wrap_content"
+                        app:cardCornerRadius="5dp"
+                        app:cardElevation="1dp"
+                        android:padding="5dp"
+                        android:layout_weight="70"
+                        android:layout_marginBottom="2dp"
+                        android:layout_marginEnd="15dp"
+                        android:visibility="visible">
+
+                        <RelativeLayout
+                            android:layout_width="match_parent"
+                            android:layout_height="match_parent">
+
+                            <TextView
+                                android:id="@+id/daysTextView"
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:textColor="@android:color/black"
+                                android:textSize="16sp"
+                                android:layout_marginStart="10dp"
+                                android:layout_marginTop="5dp" />
+
+                            <TextView
+                                android:id="@+id/daysBeforeTextView"
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:textSize="16sp"
+                                android:textColor="@android:color/black"
```

Comment:
This should probably be something like
```
android:textColor="?android:attr/textColorSecondary"
```
so it doesn't look all wrong:
![image](https://user-images.githubusercontent.com/1885159/141515867-5e0a4fd0-ff7e-48a4-88b6-44fd5cde83b2.png)

(Or maybe you don't need to set textColor at all, needs testing)

Reply:
It will look like this, if I use (android:textColor="?android:attr/textColorSecondary")
and also if I don't set textColor 
![Uploading Screenshot 2021-11-13 at 3.07.16 PM.png…]()
.

Reply:
<img width="373" alt="Screenshot 2021-11-13 at 3 07 16 PM" src="https://user-images.githubusercontent.com/68068357/141629792-32f20ed1-62a9-434c-ba62-a199c87107c2.png">

It will look like this ,If I use (android:textColor="?android:attr/textColorSecondary") and also if I don't set textColor